build.sbt

```diff
@@ -464,11 +464,30 @@ lazy val core = (project in file("modules/core"))
     )
   )
 
+lazy val currencyShared = (project in file("modules/currency-shared"))
+  .enablePlugins(AshScriptPlugin)
+  .enablePlugins(BuildInfoPlugin)
```

Comment:
Do we need BuildInfo there?

Reply:
Also add `currencyShared` to `root` so scalafix works

Reply:
I guess we add buildInfo everywhere so I did the same here.

Reply:
Not true actually. Removed the plugins that are not required at least right now.